packages/expo-image-picker/android/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,15 @@
-
-<manifest package="expo.modules.imagepicker">
-
+<manifest package="expo.modules.imagepicker"
+          xmlns:android="http://schemas.android.com/apk/res/android"
+    >
+    <application>
+        <provider
+            android:name="android.support.v4.content.FileProvider"
```

Comment:
I believe it solves the issue only because we compile and bundle all the modules into one client.

As far as I understand, if someone will use f.ex Mial Composer Module in managed flow, and for some reason ImagePickerModule will not be used and its manifest not merged into final APK, this will cause crash.

I believe the best solution would be ot define FileProvider in each module that uses it. Those should be separate providers with separate authorities but with the same paths.